floodviz/hydrograph_utils.py

```diff
@@ -2,12 +2,12 @@
 import time
 import requests
 
-
 def parse_hydrodata(jdata):
     
     """ 
     Parses json Hydrodata from NWIS webservice
-    and formats for NVD3 charting library.
+    and formats for NVD3 charting library. Upon failure, this will
+    return empty data list.
```

Comment:
I choose to use an empty list for fault returns because an empty list sparks a nice message within NVD3 that shows 'No data Available'. None type does not do this.